zip.go

```diff
@@ -93,10 +93,11 @@ func zipFile(w *zip.Writer, source string) error {
 
 		if baseDir != "" {
 			name, err := filepath.Rel(source, fpath)
+			name = strings.Replace(name, string(os.PathSeparator), "/", -1)
 			if err != nil {
```

Comment:
`filepath.Rel` returns `name` and `err`. It makes sense to check `err != nil` before proceeding.

So, you shouldn't insert the `name = ...` line here, it should be after `if err != nil { return err }` block.